packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -217,10 +217,10 @@ Media.createMediaConnection = (
     {
       // TODO: RoapMediaConnection is not ready to use stream classes yet, so we pass the raw MediaStreamTrack for now SPARK-460530
       localTracks: {
-        audio: audioStream?.outputTrack,
-        video: videoStream?.outputTrack,
-        screenShareVideo: shareVideoStream?.outputTrack,
-        screenShareAudio: shareAudioStream?.outputTrack, // TODO: add type for screenShareAudio in internal-media-core SPARK-446923
+        audio: audioStream?.outputStream.getTracks()[0],
```

Comment:
like I said in the other PR, it would be better fix webrtc-core and have `outputTrack` as public instead of protected

Reply:
Quoting 1:1 message from @brycetham :

> But regarding accessing the outputTrack in transcoded meetings, we deliberately changed webrtc-core from track-based to stream-based because it's better to use streams, since you just need to set srcObject = stream and any modifications made to the stream (like adding an effect) should work automatically. So there shouldn't be any need for accessing outputTrack directly.

and another,

> Ok, for now, if you really want to use the outputTrack, you can do stream.outputStream.getTracks()[0], with the understanding that it's a temporary workaround until transcoded meetings switches to stream classes. And actually, I thought we did do this workaround when we worked on the WCME 3.0 changes, so I'm a bit surprised to see this isn't the case in the beta code. But maybe I'm misremembering.

Reply:
@marcin-bazyl let's fix this as part of another PR after @brycetham fixes the webrtc-core for the above change.
We'd like to get this PR merged so we can run the E2E for the web client with `next`

Reply:
@marcin-bazyl @sreenara I'd really like to avoid exposing `outputTrack`, as it's not intended to be exposed as part of the change from track-based classes to stream-based classes in webrtc-core, and it risks having the track modified in ways we don't expect. Ideally, transcoded meetings should be using the stream-based classes as intended, so passing the track here is a temporary workaround, and I'd rather not change webrtc-core for something like this.

Reply:
ok, fair enough, it is true that we want to move transcoded API of internal-media-core to use the stream based classes, so let's go ahead with this as a temporary workaroud